pkg/config/config.go

```diff
@@ -1 +1,22 @@
 package config
+
+type DaemonConfig struct {
+	PeriodicUpdate int `json:"periodic_update"`
+	GuidPool       GuidPoolConfig
+	SubnetManager  SubnetManagerPluginConfig
+}
+
+type GuidPoolConfig struct {
+	GuidRangeStart string `json:"guid_range_start"`
+	GuidRangeEnd   string `json:"guid_range_end"`
+}
+
+type SubnetManagerPluginConfig struct {
+	Name        string `json:"subnet_manager_name"`
```

Comment:
it better to hold each plugin config in it own struct.
So you will have UFM config for now in the future there can be more.
so 
type SubnetManagerPluginConfig struct {
UFMConfig
}
in the future :
type SubnetManagerPluginConfig struct {
UFMConfig
OtherPlugin
}

Reply:
> ixed